roles/atomic_scan_verify/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+# vim: set ft=ansible:
+#
+#   This role verifies the atomic scan command.  It validates the command by
+#   checking the output of the command and looking for the log file generated
+#   by the scan command.
+#
+#   Commands:
+#   - atomic install
+#   - atomic scan
+#
+
+- name: Install openscap
+  command: atomic install rhel7/openscap
+  register: result
+
+- name: Check installation of openscap
+  fail:
+    msg="Atomic install was unsuccessful"
+  when: "'Installation complete' not in result.stdout"
+
+- name: Run atomic scan
+  shell: python -c "import pty; pty.spawn(['atomic', 'scan', '--scanner', 'openscap', 'rhel7/openscap'])"
+  register: atomic_output
+
+- name: Catch atomic scan failures
+  fail:
+    msg="Atomic scan failed"
+  when: "'Files associated' not in atomic_output.stdout"
+
+- name: Get scanner result directory
+  shell: echo "{{ atomic_output.stdout }}" | awk '/^Files associated/{print $8}' | sed 's/\.//'
```

Comment:
You should use the `quote` filter when passing variables to the `shell:` module

https://github.com/openshift/openshift-ansible/blob/master/docs/best_practices_guide.adoc#The-Ansible-quote-filter-MUST-be-used-with-any-variable-passed-into-the-shell-module